code/game/machinery/computer/crew.dm

```diff
@@ -33,9 +33,17 @@
 		src.updateDialog()
 		return
 
+var/datum/singleton/crewmonitor/crewmonitor = new

```

Comment:
"Singleton" is a generic word used to reference a type of object that only exists once in the entire project (as is the case here).

Relevant wiki entry: https://en.wikipedia.org/wiki/Singleton_variable

It doesn't seem to be confined to specific languages. In fact, the word is used all the time in popular Java libraries even though Java doesn't support anything to enforce singletons.

The reason I added it in this case is so that a developer will immediately know that there will only be one object of that type, with the ulterior motive of making the code more readable.